### DISCUSSION TITLE
Login with e-mail address is case-sensitive

### DISCUSSION THREAD

**Discussion by jannislehmann** [requarks/wiki#3528] [Error / Bug Report]
**Describe the bug**
When trying to log in using an e-mail address, the e-mail has to be case-senstive.

**To Reproduce**
1. Create a user with an e-mail
2. Try to log in with any character in uppercase.

**Expected behavior**
The e-mail address should be handled case-insenstive.

**Comment by vladisalv** [comment]
++

Also, please, strip spaces in e-mail

**Comment by JetForMe** [comment]
This is a regular problem for me, as users on mobile generally end up with the first letter auto-capitalized. I'll open a separate bug about setting the field type.
